boards/arm/ev11l78a/ev11l78a-pinctrl.dtsi

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2023, Meta Platforms, Inc. and its affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/samd20eXab-pinctrl.h>
+
+&pinctrl {
+	sercom0_spi_default: sercom0_spi_default {
+		group1 {
+			pinmux = <PA8C_SERCOM0_PAD0>,
+				 <PA9C_SERCOM0_PAD1>,
+				 <PA11C_SERCOM0_PAD3>;
+		};
+	};
```

Comment:
Is this right? It looks disconnected from the schematic you linked in the docs.

Reply:
Oh it's a chiplet! Cool, sorry did not realize, you even mentioned it in the docs. Good stuff thanks for clarifying.